Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/TaitSurfacePressureForceField.inl

```diff
@@ -396,6 +398,64 @@ void TaitSurfacePressureForceField<DataTypes>::addKToMatrixT(const core::Mechani
     }
 }
 
+template <class DataTypes>
+void TaitSurfacePressureForceField<DataTypes>::buildStiffnessMatrix(sofa::core::behavior::StiffnessMatrix* matrix)
```

Comment:
don't know if this is relevant but I see you copy past the content of the method addKToMatrixT and not addKToMatrix like what you did in other components.

Reply:
You are right. In this force field, `addKToMatrix` calls `addKToMatrixT`, but it is hidden by several layers (`BlocMatrixWriter`). See https://github.com/sofa-framework/sofa/blob/e4b94df31793bbe4b8ef62dd7412f26b6a07d18f/Sofa/framework/Core/src/sofa/core/behavior/BlocMatrixWriter.h#L157